ajax/libs/jquery.socialshareprivacy/package.json

```diff
@@ -8,5 +8,10 @@
     "privacy",
     "datenschutz",
     "heise"
-  ]
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/patrickheck/socialshareprivacy.git"
```

Comment:
According to #709 and https://www.heise.de/extras/socialshareprivacy/, I believe that this repo is just a fork of it, it's not the original one.